citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/ListAuthoritiesController.java

```diff
@@ -0,0 +1,83 @@
+package edu.asu.diging.citesphere.web.user.authorities;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import edu.asu.diging.citesphere.core.exceptions.GroupDoesNotExistException;
+import edu.asu.diging.citesphere.core.exceptions.ZoteroHttpStatusException;
+import edu.asu.diging.citesphere.core.service.IAuthorityService;
+import edu.asu.diging.citesphere.core.service.ICitationManager;
+import edu.asu.diging.citesphere.model.authority.IAuthorityEntry;
+import edu.asu.diging.citesphere.model.bib.ICitation;
+import edu.asu.diging.citesphere.model.bib.ICitationGroup;
+import edu.asu.diging.citesphere.model.bib.ICreator;
+import edu.asu.diging.citesphere.model.bib.IPerson;
+import edu.asu.diging.citesphere.model.bib.impl.CitationResults;
+import edu.asu.diging.citesphere.user.IUser;
+
+@Controller
+public class ListAuthoritiesController {
+
+    @Autowired
+    private IAuthorityService authorityService;
+
+    @Autowired
+    private ICitationManager citationManager;
+
+    @RequestMapping("/auth/authority/{authorityId}/listByName")
```

Comment:
Camel case for pathvariable is ok. But camel case as part of the API end-point is not a standard practice. Referring to "listByName".